src/components/LeftDrawerOrg/LeftDrawerOrg.tsx

```diff
@@ -183,16 +174,21 @@ const leftDrawerOrg = ({
                 {({ isActive }) => (
                   <Button
                     key={name}
-                    variant={isActive ? 'success' : ''}
-                    className={
-                      isActive ? styles.activeButton : styles.inactiveButton
-                    }
+                    variant={isActive === true ? 'success' : ''}
+                    style={{
+                      backgroundColor: isActive === true ? '#EAEBEF' : '',
```

Comment:
1. Don't use hard coded colors
1. Make this file use CSS exclusively from `src/style/app.module.css`, with the exception of package based CSS.
1. Delete the CSS file in the directory of this file. Make the file exclusively use CSS from `src/style/app.module.css`
1. Use no CSS in `assets/css/app.css`
1. This will help with future attempts to having Dark Mode
1. Use a variable from `src/style/app.module.css` .
1. Use a variable name that describes the function of the color and not the color itself. For example `--search-button-bg` is much better than `--light-blue` 

- https://github.com/PalisadoesFoundation/talawa-admin/issues/3296